src/shared/components/FormField/FormField.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { ChildrenWrapper, FormFieldDescription, FormFieldTitle, FormFieldWrapper } from './FormField.style'
+
+export type FormFieldProps = {
+  title: string
+  description?: string | string[]
```

Comment:
Just an idea - we could always pass an array of strings so there would be no need for ternary operator or type guard in the component - `description` could be mapped even if it has only one string. What do you think? I would still consider that interface friendly

Reply:
I like how it is set up right now. It's more intuitive to pass a string here in the first place. 

I would think about passing an array of strings as an extra option. Still not sure if I should keep it. I just got an idea that can be useful sometimes.